modules/manual-configuration-image-registry-gar.adoc

```diff
@@ -1,27 +1,29 @@
 // Module included in the following assemblies:
 //
 // * integration/integrate-with-image-registries.adoc
+
 :_mod-docs-content-type: PROCEDURE
 [id="manual-configuration-image-registry-gar_{context}"]
 = Manually configuring Google Artifact Registry
 
-You can integrate {product-title} with Google Artifact Registry.
+You can integrate {rh-rhacs-first} with the Google Artifact Registry.
 
 .Prerequisites
-* You need either a link:https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[workload identity] or a service account key for authentication.
-* The associated service account must have the *Artifact Registry Reader* Identity
-and Access Management (IAM) role `roles/artifactregistry.reader`.
+* You have a workload identity or service account key for authentication.
++
+For more information, see link:https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Authenticate to Google Cloud APIs from GKE workloads] (Google Cloud documentation).
+* You have the artifact registry reader Identity
+and Access Management (IAM) role, `roles/artifactregistry.reader` for the associated service account.
 
 .Procedure
-. In the {product-title-short} portal, go to *Platform Configuration* -> *Integrations*.
-. Under the *Image Integrations* section, select *Google Artifact Registry*.
-. Click *New integration*.
-. Enter the details for the following fields:
-.. *Integration name*: The name of the integration.
-.. *Registry endpoint*: The address of the registry.
-.. *Project*: The Google Cloud project name.
-.. *Use workload identity*: Check to authenticate using a workload identity.
-.. *Service account key (JSON)*: Your service account key for authentication.
-. Select *Create integration without testing* to create the integration without testing the connection to the registry.
-. Select *Test* to test that the integration with the selected registry is working.
-. Select *Save*.
+. In the {product-title-short} portal, click *Platform Configuration* -> *Integrations*.
+. In the *Image Integrations* section, click *Google Artifact Registry*.
+. To create a new integration, click *New integration*.
+. Enter the name for your integration. You can save your integration only after you specify a name for your integration.
```

Comment:
All of these "You can save your integration only after you specify..." seem a bit repetitive and unnecessary.

I assume that there are client-side checks that don't let you hit *Save* or something until these are filled in.

Since all of the other steps that aren't required start with "Optional:", it should be clear that these three that aren't marked optional are required. Recommend removing these "You can save your integration only after you specify" sentences from the 3 steps.